doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+Expected result: The nodes are spread across the canvas.
 
-Actual result: As user B, you cannot see the imported project in the space.
+Actual result: All nodes are in the same position.
 
-Workaround: As user B, reload the space.
+Workaround: Click the image:ReleaseNotes/automatic_layout.png[] *Automatic layout* button in the toolbar.
 
-//approved
-.In a high-availability authoring environment, role changes in {CENTRAL} are not saved [https://issues.jboss.org/browse/RHPAM-2342[RHPAM-2342]]
 
-Issue: If you configure a role change in high-availability {CENTRAL}, the change can be lost after a time and the roles reset to defaults.
+ifdef::PAM[]
 
-Workaround: Configure the role change again.
+== Process designer
 
-//approved
-.In a high-availability authoring environment, an imported project in {CENTRAL} appears multiple times [https://issues.jboss.org/browse/RHPAM-2372[RHPAM-2372]]
+.If the properties panel is opened, after you maximize and restore the panel, it closes [https://issues.redhat.com/browse/RHPAM-2613[RHPAM-2613]]
 
-Issue: If you import a sample project in high-availability {CENTRAL}, the project can appear in the list several times. Some copies might not have all of the assets. The environment can slow down when this issue occurs.
+Issue: If the properties panel for a process is open and then you maximize and restore the panel, it closes.
 
-Workaround: Ensure one copy has all of the assets, then remove all other copies. If {CENTRAL} performance is slow, restart all of the nodes connected to it.
+Steps to reproduce:
 
-//approved
-.In a high-availability authoring environment, project creation fails when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2475[RHPAM-2475]]
+. Create a process.
+. Open the properties panel for the process.
+. Click the image:ReleaseNotes/maximize-button.png[] *Maximize* button to maximize the panel to the size of the entire screen.
+. Click the image:ReleaseNotes/minimize-button.png[] *Maximize* button to restore the panel to its previous size.
 
-Issue: When you create a project, the project creation does not complete. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+Expected result: The properties panel remains open.
 
-Workaround: Create the project again.
+Actual result: The properties panel closes.
 
-//approved
-.In a high-availability authoring environment, asset creation is not completed when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2476[RHPAM-2476]]
+Workaround: Open the properties panel again.
 
-Issue: When you create an asset, the asset is not indexed. It is displayed in {CENTRAL} but you cannot use it. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+.If the properties panel is opened, after you switch to a different view, the panel closes [https://issues.redhat.com/browse/RHPAM-2612[RHPAM-2612]]
```

Comment:
If the process designer *Properties* panel is open and you switch to a different view, the panel closes